### PR TITLE
Designed view for nereid passbook #7319

### PR DESCRIPTION
**Pull request by vishu1993**

### CODE REVIEW

**Comment by PritishC** [comment]
Squash commits into single commit.

File: passbook.xml [hunk 1]
```diff
@@ -1,5 +1,49 @@
 <?xml version="1.0"?>
 <tryton>
     <data>
+        <menuitem name="PassBook" id="menu_passbook" parent="nereid.menu_nereid" sequence="10" icon="tryton-list"/>
+        <menuitem name="Passes" id="menu_passes"  parent="menu_passbook" action="act_nereid_passbook_form" sequence="10" icon="tryton-list"/>
```
Comment (sharoonthomas) [inline passbook.xml]: the action is declared before the window. This wont work.

**Comment by vishu1993** [comment]
passbook

File: view/registration_tree_sequence.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<tree string="Registrations" sequence="sequence" >
+    <field name="device_library_identifier" />
+    <field name="push_token" />
+</tree>
+
```
Comment (sharoonthomas) [inline view/registration_tree_sequence.xml]: remove

File: view/nereid_passbook_form.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    <label name="active" />
+    <field name="active" />
+    <label name="origin" />
+    <field name="origin" />
+    <newline/>
+    <label name="last_update" />
+    <field name="last_update"/> 
+    <label name="authentication_token" />
+    <field name="authentication_token" />  
+    <field name="registrations" colspan="4" view_ids="nereid_passbook.nereid_passbook_registration_view_tree_sequence"/>
```
Comment (sharoonthomas) [inline view/nereid_passbook_form.xml]: I dont see the need for the views to be spceified

**Comment by vishu1993** [comment]
nereid-passbook

File: passbook.xml [hunk 4]
```diff
@@ -1,5 +1,48 @@
...
+            <field name="type">form</field>
+            <field name="name">registration_form</field>
+        </record>
+        <record model="ir.ui.view" id="nereid_passbook_registration_view_tree">
+            <field name="model">nereid.passbook.registration</field>
+            <field name="type">tree</field>
+            <field name="priority" eval="10"/>
+            <field name="name">registration_tree</field>
+        </record>
+        <record model="ir.ui.view" id="nereid_passbook_registration_view_tree_sequence">
```
Comment (sharoonthomas) [inline passbook.xml]: I dont see the need for this view

File: passbook.xml [hunk 5]
```diff
@@ -1,5 +1,48 @@
...
+            <field name="model">nereid.passbook.registration</field>
+            <field name="type">form</field>
+            <field name="name">registration_form</field>
+        </record>
+        <record model="ir.ui.view" id="nereid_passbook_registration_view_tree">
+            <field name="model">nereid.passbook.registration</field>
+            <field name="type">tree</field>
+            <field name="priority" eval="10"/>
+            <field name="name">registration_tree</field>
+        </record>
```
Comment (sharoonthomas) [inline passbook.xml]: Bind the above two views to windows

File: view/nereid_passbook_form.xml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<form string="Nereid Passbook" col="4">
+    <label name="active" />
+    <field name="active" />
+    <label name="origin" />
```
Comment (sharoonthomas) [inline view/nereid_passbook_form.xml:3]: origin before the active checkbox

**Comment by vishu1993** [comment]
nereidpassbook

**Comment by vishu1993** [comment]
nereidpassbookfinal

**Comment by PritishC** [comment]
LGTM.

File: passbook.xml [hunk 7]
```diff
@@ -1,5 +1,31 @@
...
+            <field name="view" ref="nereid_passbook_view_tree" />
+            <field name="act_window" ref="act_nereid_passbook_form" />
+        </record>
+        <record model="ir.action.act_window.view" id="act_nereid_passbook_form_view2">
+            <field name="sequence" eval="20" />
+            <field name="view" ref="nereid_passbook_view_form" />
+            <field name="act_window" ref="act_nereid_passbook_form" />
+        </record>
+        <menuitem name="PassBook" id="menu_passbook" parent="nereid.menu_nereid" sequence="10" icon="tryton-list"/>
+        <menuitem name="Passes" id="menu_passes"  parent="menu_passbook" action="act_nereid_passbook_form" sequence="10" icon="tryton-list"/>
```
Comment (sharoonthomas) [inline passbook.xml]: make this multiple lines and < 80 chars

**Comment by PritishC** [comment]
LGTM.

**Pull request merged**